### PR TITLE
feat(HACBS-1253): enhance malformed json testing

### PR DESCRIPTION
**Pull request by Josh-Everett**
Additional test that includes data intended to mimic a broken json file.
Signed-off-by: Josh Everett <jeverett@redhat.com>

### CODE REVIEW

File: helpers/integration_test.go [hunk 1]
```diff
@@ -340,6 +341,38 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
...
+			PipelineRunStatusFields: tektonv1beta1.PipelineRunStatusFields{
+				TaskRuns: map[string]*tektonv1beta1.PipelineRunTaskRunStatus{
+					"task1": {
+						PipelineTaskName: "task-malformed-result",
+						Status: &tektonv1beta1.TaskRunStatus{
+							TaskRunStatusFields: tektonv1beta1.TaskRunStatusFields{
+								TaskRunResults: []tektonv1beta1.TaskRunResult{
+									{
+										Name:  "HACBS_TEST_OUTPUT",
+										Value: overflow.String(),
```
Comment (dirgim) [inline helpers/integration_test.go:360]: Something like `Value: *tektonv1beta1.NewArrayOrString(overflow.String())` should work here.

File: helpers/integration_test.go [hunk 2]
```diff
@@ -340,6 +341,38 @@ var _ = Describe("Pipeline Adapter", Ordered, func() {
...
+							TaskRunStatusFields: tektonv1beta1.TaskRunStatusFields{
+								TaskRunResults: []tektonv1beta1.TaskRunResult{
+									{
+										Name:  "HACBS_TEST_OUTPUT",
+										Value: *tektonv1beta1.NewArrayOrString(`{
+											"success":false,
+											"errors":[{"code":6007,"message":"Malformed JSON in request body"}],
+											"messages":[],
+											"result":null
+										}`)
```
Comment (dirgim) [inline helpers/integration_test.go:360]: Looks like a comma is missing here:
```
Error: helpers/integration_test.go:360:14: missing ',' before newline in composite literal
```

Comment (Josh-Everett) [reply helpers/integration_test.go]: Changed the formatting in a way I think removes some of the visual ambiguity here - I kept glancing right over this.

**Review by dirgim — APPROVED** [review]
LGTM

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**